test/libsolidity/semanticTests/externalSource/source_remapping_absolute.sol

```diff
@@ -0,0 +1,15 @@
+==== ExternalSource: /ExtSource.sol=_external/external.sol ====
```

Comment:
Some test ideas. I'm listing files and the `ExternalSources`/`Sources`/`import`s they should contain for brevity:
1) Relative imports in external sources:
    - `semanticTests/externalSource/relative_imports.sol`
        - `ExternalSource: _relative_imports/contract.sol`
        - `ExternalSource: _relative_imports/dir/a.sol`
        - `ExternalSource: _relative_imports/dir/B/b.sol`
        - `ExternalSource: _relative_imports/c.sol`
        - `ExternalSource: _relative_imports/D/d.sol`
        - `ExternalSource: _relative_imports/G/g.sol`
        - `ExternalSource: _relative_imports/h.sol`4
        - `import "_relative_imports/contract.sol"`
    - `semanticTests/externalSource/_relative_imports/contract.sol`
        - `import "./a.sol" as a`
        - `import "./B/b.sol" as b`
        - `import "../c.sol" as c`
        - `import "../D/d.sol" as d`
        - `import "./E/../F/../G/./g.sol" as g`
        - `import "../../../../_relative_imports/h.sol" as h`
    - `semanticTests/externalSource/_relative_imports/dir/a.sol`
    - `semanticTests/externalSource/_relative_imports/dir/B/b.sol`
        - `import "../c.sol"`
    - `semanticTests/externalSource/_relative_imports/c.sol`
    - `semanticTests/externalSource/_relative_imports/D/d.sol`
    - `semanticTests/externalSource/_relative_imports/G/g.sol`
        - `import "../B/b.sol"`
    - `semanticTests/externalSource/_relative_imports/h.sol`
2) Duplicate external sources
    - `semanticTests/externalSource/_duplicate_external_sources.sol`
        - `ExternalSource: _duplicate_external_sources/contract.sol`
        - `ExternalSource: _duplicate_external_sources/contract.sol`
        - `ExternalSource: contract.sol=_duplicate_external_sources/contract.sol`
        - `ExternalSource: contract.sol=_duplicate_external_sources/contract.sol`
        - `Source: _duplicate_external_sources/contract.sol`
        - `Source: contract.sol`
        - `import "_duplicate_external_sources/contract.sol" as contract1`
        - `import "contract.sol" as contract2`
    - `semanticTests/externalSource/_duplicate_external_sources/contract.sol`
    - Contracts should contain something that will make it clear whether it's the `Source` or `ExternalSource` that ends up being imported in this situation.
3) External sources with non-normalized paths
    - `semanticTests/externalSource/_non_normalized_paths.sol`
        - `ExternalSource: _non_normalized_paths/a.sol`
        - `ExternalSource: _non_normalized_paths//a.sol`
        - `ExternalSource: _non_normalized_paths//b.sol`
        - `ExternalSource: C/c.sol=_non_normalized_paths/c1.sol`
        - `ExternalSource: C//c.sol=_non_normalized_paths/c2.sol`
        - `ExternalSource: D//d.sol=_non_normalized_paths/c2.sol`
        - `import "_non_normalized_paths//a.sol"`
        - `import "_non_normalized_paths//b.sol"`
        - `import "C//c.sol"`
        - `import "D//d.sol"`
4) Remappings with multiple `=` signs:
    - `semanticTests/externalSource/multiple_equals_signs.sol`
        - `ExternalSource: a=_multiple_equals_signs/contract.sol=c=d.sol`
        - `import "a"`
    - `semanticTests/externalSource/_multiple_equals_signs/contract.sol=c=d.sol`
5) External files importing files from a multi-source file:
    - `semanticTests/externalSource/_importing_from_multisource.sol`
        - `ExternalSource: _importing_from_multisource/contract.sol`
        - `Source: a.sol`
        - `import "_importing_from_multisource/contract.sol"`
    - `semanticTests/externalSource/_importing_from_multisource/contract.sol`
        - `import "a.sol"`
6) External source names containing `./` and `../`
    - `semanticTests/externalSource/_source_name_starting_with_dots.sol`
        - `ExternalSource: _source_name_starting_with_dots/dir/contract.sol`
        - `ExternalSource: _source_name_starting_with_dots/dir/a.sol`
        - `ExternalSource: _source_name_starting_with_dots/b.sol`
        - `ExternalSource: ./a.sol=_source_name_starting_with_dots/dot_a.sol`
        - `ExternalSource: ../b.sol=_source_name_starting_with_dots/dot_dot_b.sol`
        - `Source: ./a.sol`
        - `Source: ../b.sol`
        - `import "_source_name_starting_with_dots/contract.sol"`
    - `semanticTests/externalSource/_source_name_starting_with_dots/dir/contract.sol`
        - `import "./a.sol"`
        - `import "../b.sol"`
    - `semanticTests/externalSource/_source_name_starting_with_dots/dir/a.sol`
    - `semanticTests/externalSource/_source_name_starting_with_dots/b.sol`
    - `semanticTests/externalSource/_source_name_starting_with_dots/dot_a.sol`
    - `semanticTests/externalSource/_source_name_starting_with_dots/dot_dot_b.sol`
    - Contracts should contain something that will make it clear which of these files end up being imported.